supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -1,3 +1,37 @@
+[discrete]
+[[AArch64]]
+==== image:images/yes.png[yes] AArch64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term when referring to Operating systems and server instances. Including: Linux versions, RHEL, or CoreOS. Use this format in general cases when refering to system architecture. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands or outputs, confer with your SME on the correct format for the specific use case.
+
+Example:
+
+_When running RHEL with an AArch64 system, run the following command._
```

Comment:
_When running RHEL with an AArch64 system, run the following command._ > _When running RHEL with an AArch64 system, run the following command:_